elastic/kibana
# [AO] Remove extra fields such as cpu, memory, ... from context.container action variable and AAD

Closes #161267

## 📝 Summary 

This PR removes CPU, memory, disk, or network values from AAD and action context variables. I will add a test for this implementation in another [PR](https://github.com/elastic/kibana/pull/161569).

| Before | After |
|---|---|
|![image](https://github.com/elastic/kibana/assets/12370520/5050a0bf-5780-48ed-ad62-2d40145842ba)|![image](https://github.com/elastic/kibana/assets/12370520/38a25ac7-c656-42d0-ac6f-9821b7614d3a)|


## 🧪 How to test
- Create a new threshold rule and add an action with the `context.container` action variable
- Make sure you set the group to `kubernetes.pod.uid`
- Check the value in the triggered action, you should not see CPU, memory, disk, or network values there
